nbconvert/exporters/templateexporter.py

```diff
@@ -25,34 +25,34 @@
 JINJA_EXTENSIONS = ['jinja2.ext.loopcontrols']
 
 default_filters = {
-        'indent': filters.indent,
-        'markdown2html': filters.markdown2html,
-        'ansi2html': filters.ansi2html,
-        'filter_data_type': filters.DataTypeFilter,
-        'get_lines': filters.get_lines,
-        'highlight2html': filters.Highlight2HTML,
-        'highlight2latex': filters.Highlight2Latex,
-        'ipython2python': filters.ipython2python,
-        'posix_path': filters.posix_path,
-        'markdown2latex': filters.markdown2latex,
-        'markdown2rst': filters.markdown2rst,
```

Comment:
that's only from the default set of templates.

`markdown2html_pandoc` was also removed elsewhere.